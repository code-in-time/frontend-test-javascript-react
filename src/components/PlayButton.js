import React, { Component } from 'react';

class PlayButton extends Component {

  render() { 
    return ( <button className="btn btn-primary" onClick={this.props.onClick}>Play</button> );
  }
}
 
export default PlayButton;