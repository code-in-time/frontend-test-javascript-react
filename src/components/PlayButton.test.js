import React from 'react';
import renderer from 'react-test-renderer';
import PlayButton from './PlayButton';

it('Should test the PlayButton.js snapshot', () => {
  const tree = renderer.create(<PlayButton />).toJSON()
  expect(tree).toMatchSnapshot();
})
