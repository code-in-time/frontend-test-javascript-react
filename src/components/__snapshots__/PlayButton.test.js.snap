// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test the PlayButton.js snapshot 1`] = `
<button
  className="btn btn-primary"
>
  play
</button>
`;
